cluster/node-pools/master-default/userdata.yaml

```diff
@@ -124,6 +124,9 @@ write_files:
           - --service-account-signing-key-file=/etc/kubernetes/ssl/service-account-private-key.pem
           - --service-account-issuer=kubernetes/serviceaccount
           {{- end }}
+          - --service-account-key-file=/etc/kubernetes/ssl/pod-identity-service-account-public-key.pem
+          - --service-account-signing-key-file=/etc/kubernetes/ssl/pod-identity-service-account-key.pem
+          - --service-account-issuer=https://{{.Cluster.LocalID}}.{{.Values.hosted_zone}}/
```

Comment:
Probably makes sense to simply use the `APIServerURL` here instead of constructing it manually. And I guess the same could be done the other place you pass `hosted_zone`.

Reply:
Will be done.